app/models/user.rb

```diff
@@ -37,6 +37,10 @@ def guest?
     self.meta && self.meta.is_a?(Guest)
   end
 
+  def simple_admin?
+    self.meta_type == 'Manager' && !self.sub_admin && !self.admin
```

Comment:
i think it's better to use `self.meta_type == 'Manager' && !self.sub_admin? && !self.admin?`